core/src/main/java/io/questdb/griffin/SqlUtil.java

```diff
@@ -846,6 +883,14 @@ private static long implicitCastStrVarcharAsTimestamp0(CharSequence value, int c
         return Numbers.LONG_NULL;
     }
 
+    private static boolean isNotWrapperModel(QueryModel model) {
```

Comment:
This method returns `false` for model that includes window functions which seems wrong - we need to check `model.getSelectModelType()`'s result. Might be a good idea to add some unit tests for this method.